src/wp-admin/includes/user.php

```diff
@@ -103,24 +103,22 @@ function edit_user( $user_id = 0 ) {
 	}
 
 	/**
-	 * Enable adding of user to user taxonomies.
+	 * Enable adding of user taxonomies unless a filter is set to false.
 	 *
 	 * @since CP-2.1.0
 	 */
-	$tax_array = array();
-	$taxonomies = get_object_taxonomies( 'user', 'objects' );
-	if ( ! empty( $taxonomies ) ) {
-		foreach ( $taxonomies as $taxonomy ) {
-			$tax_array[] = $taxonomy->name;
-			foreach ( $tax_array as $tax_field ) {
-				if ( isset( $_POST[ $tax_field ] ) ) {
-					$tax_names = array();
-					foreach ( $_POST[ $tax_field ] as $tax_name ) {
-						$tax_names[] = wp_unslash( $tax_name );
+	if ( ! has_filter( 'cp_user_taxonomies', '__return_false' ) && ! has_filter( 'user_edit_taxonomies', '__return_false' ) ) {
```

Comment:
Someone can add different filters that can return false.
Anyway if this check is bypassed in that way only more PHP code lines are executed but the result is the same. So good for me.

Reply:
This line will probably still work and would perhaps make more sense on readings as follows:

`if ( false !== apply_filters( 'cp_user_taxonomies', true ) && false !== apply_filters( 'user_edit_taxonomies', true ) ) {`

Reply:
Sure. But maybe better passing all the parameters that are expected and of the expected type, even empty.